cdap-app-fabric-tests/src/test/java/io/cdap/cdap/internal/app/runtime/batch/DynamicPartitionerWithAvroTest.java

```diff
@@ -172,6 +172,7 @@ public void apply() throws IOException {
     // run the partition writer m/r with this output partition time
     Map<String, String> arguments = new HashMap<>();
     arguments.put(OUTPUT_PARTITION_KEY, Long.toString(now));
+    arguments.put("system.mapreduce.mapreduce.fileoutputcommitter.algorithm.version", "1");
```

Comment:
Add a comment here explaining why this property is set. We also need to document this in case a customer needs to overwrite this.